omnigibson/examples/objects/load_object_selector.py

```diff
@@ -48,6 +48,9 @@ def main(random_selection=False, headless=False, short_exec=False):
     }
     if scene_type == "InteractiveTraversableScene":
         cfg["scene"]["scene_model"] = "Rs_int"
+    else:
+        cfg["scene"]["floor_plane_grid"] = True
+        cfg["scene"]["use_skybox_texture"] = False
```

Comment:
should we keep the original behavior as default? I think the skybox actually looks better than the checkerboard, but I guess "better" is also subjective.

Reply:
This example is the whole point of adding the checkerboard, it lets you see the object regardless of its color, get a feel for its size, and also not have the object floating around in front of some clouds. This is the only example that I changed.

Reply:
FYI, slightly unrelated, but we should probably update the default skybox to be an actual HDRI background from [here](https://polyhaven.com/hdris). They're free to use for non-commerical purposes, are incredibly high quality (can go up to 8k), and have nuanced lighting / toning baked in based on the backdrop. Was experimenting with them for our digital cousin project and it's literally a drop in replacement for our current janky skybox that I had originally added to OG lol (literally just replace via `og.sim.skybox.texture_path = ...`)

Reply:
I tried some of these the other day (from this exact website) - it's kind of hard to get this right. The problem is that many scenes have full height windows, and what you see outside through them makes the scene look very weird regardless of it's the sky both up and down, or grass and sky. I think the ideal solution involves using some kind of backdrop city mesh, which I will look into later.

Reply:
Resolving for now, opening a ticket for this.